app/resources/services/geoip.js

```diff
@@ -0,0 +1,65 @@
+const request = require('request')
+const redis = require('redis')
+const config = require('config')
+const logger = require('../../../lib/logger.js')()
+
+const IP_GEOLOCATION_TIMEOUT = 500
+
+exports.get = function (req, res) {
+  if (req.headers.host !== config.app_host_port) {
+    res.status(403).json({ status: 403, error: 'Not allowed to access API' })
+    return
+  }
+
+  const requestIp = function (req) {
```

Comment:
Since this is the same as the one in (https://codeclimate.com/github/streetmix/streetmix/app/resources/v1/streets.js?from_sha=c2803e09&to_sha=1f83f2ed) (good find on this btw) I think we should break this function out (maybe put it in `lib/util.js`?) and import it into both files.